README.md

```diff
@@ -3,6 +3,39 @@
 ## Overview
 
 A lightweight wrapper around [{EpiNow2}](https://github.com/epiforecasts/EpiNow2) to add functionality for deployment in Azure Batch.
+It holds some helper functions to interface with Azure services, convert input data to EpiNow2's expected input format, and save expected outputs. 
+It also adds metadata and logging.
+
+This package is meant to enhance the `{EpiNow2}` package to support deployment in CFA's computational environment.
+The code is open source as part of CFA's goals around development, but it may not be possible to support extensions to additional environments.
+
+## Structure
+
+This repository holds an R package, `{CFAEpiNow2Pipeline}`.
+The repository is structured as a standard R package.
+All PRs pass R CMD check as part of the CI suite as a pre-condition for merge to main.
+If interested in contributing see `CONTRIBUTING.md` and open an issue or a PR.
+
+The package contains contains some adapters and wrappers to run to run many independent `{EpiNow2}` models in parallel with cloud resources.
+The adapters read from datasets with standardized formats and produces outputs as flat files with standard names.
+The wrapper functions enhance EpiNow2 functionality to support cloud deployments, adding more logging and standardizing the R environment.
+
+This package standardizes the interface to EpiNow2 for purposes of deployment in a pipeline as part of a suite of models.
+This package does _not_ manage pipeline deployment or kickoff, data extraction and transformation, or model output visualization.
+
+## Components
+
+This package implements functions for:
+
+1. **Configuration**: Loads parameters such as prior distributions, generation intervals, and right-truncation from a config in a standard schema, with the path to this config passed at runtime.
+    - The config is validated at runtime, but config generation is specified at pipeline runtime and not part of this package.
+1. **Data load**: Loads data from the CFA data lake or from a local environment and translates it from CFA's schema to the expected EpiNow2 format.
+    - Paths are specified via the config
+1. **Parameters**: Loads pre-specified and -validated generation interval, delay interval, and right-truncation distributions from from the CFA data lake or from a local environment and formats them for use in EpiNow2.
+1. **Model run**: Manages R environment to run EpiNow2 from a fixed random seed, both for `{EpiNow2}` initialization and Stan sampling.
+1. **Outputs**: Provides functionality to process `{EpiNow2}` model fits to a standardised flat output format (as described in forthcoming link). Within the pipeline, model fits are saved both in their entirety as `.rds` files, as well as via this flat output format.
+1. **Logging**: Steps in the pipeline have comprehensive R-style logging, with the the [cli](https://github.com/r-lib/cli) package
```

Comment:
Arguably this seems like more of a feature of the functions in the package rather than a function itself.

Are there functions which do specific logging in this package? Or is the logging a feature of each of the other functions?

Reply:
The logging is embedded in the other functions. I think it's important to call out because not having it has been a pain point even if it deviates from the pattern of the list a bit.